packages/widget/src/controllers/wallet-manager/manager.ts

```diff
@@ -7,15 +7,40 @@ import Onboard from '@web3-onboard/core';
 import injectedModule from '@web3-onboard/injected-wallets';
 import walletConnectModule from '@web3-onboard/walletconnect';
 import type { ReactiveController, ReactiveElement } from 'lit';
+import type { WalletInit } from '@web3-onboard/common';
 
 import { utils } from 'ethers';
 import { WalletUpdateEvent, walletContext } from '../../context';
 
 export class WalletController implements ReactiveController {
+  readonly WALLET_CONNECT_PROJECT_ID = '2f5a3439ef861e2a3959d85afcd32d06';
```

Comment:
should this be hardcoded?  or did you forget to delete it?

Reply:
@wainola I have used it from [here](https://github.com/sygmaprotocol/sygma-widget/blob/13235bb6784e2d0e6249d30aa58c1beef4068358/packages/widget/src/controllers/wallet-manager/manager.ts#L101C23-L101C55)